text/0000-optional-jquery.md

```diff
@@ -0,0 +1,280 @@
+- Start Date: 2018-01-11
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Make jQuery optional
+
+## Summary
+
+For the past Ember has been relying and depending on jQuery. This RFC proposes making jQuery optional and having a well 
+defined way for users to opt-out of bundling jQuery. 
+
+## Motivation
+
+### Why we don't need jQuery any more
+
+One of the early goals of jQuery was cross-browser normalization, at a time where browser support for web standards was 
+incomplete and inconsistent, and Internet Explorer 6 was the dominating browser. It provided a useful and convenient
+API for DOM traversal and manipulation as well as event handling, that hid the various browser differences and bugs from
+the user. For example `document.querySelector` wasn't a thing at that time, and browsers were using very different event 
+models ([DOM Level 0, DOM Level 2 and IE's own proprietary model](https://en.wikipedia.org/wiki/DOM_events#Event_handling_models)). 
+
+But this level of browser normalization is not required anymore, as today's browsers all support the basic DOM APIs well
+enough. Even more so that the upcoming Ember 3.0 will drop support for all versions of Internet Explorer except 11.
+
+Furthermore Ember users will need to directly traverse and modify the DOM or manually attach event listeners in very 
+special cases only. Most of these low level interactions are taken care of by Ember's templates and its underlying 
+Glimmer rendering engine, as well as action helpers or the component's event handler methods.
+
+So having jQuery included by default does not provide that much value to users most of the time, and Ember itself is
+expected to be fully functional and tested without jQuery, presumably for the upcoming 3.0 stable release.
+
+### What are the drawbacks of bundling jQuery
+
+The major drawback is the increased bundle size, which amounts to [~29KB](https://mathiasbynens.be/demo/jquery-size) 
+(minified and gzipped). This not only increases the loading time, but also parse and compile times, thus increasing the 
+total time to interactive. This is especially true for mobile devices, where slow connectivity and weak CPU performance 
+is not uncommon.
+
+Having jQuery not included will improve the suitability of Ember for mobile applications considerably. Even
+if the raw number is not that huge, it all adds up. And it plays together with other efforts to make leaner Ember builds
+possible, like enabling tree shaking with the new [Module API](https://github.com/emberjs/rfcs/blob/master/text/0176-javascript-module-api.md),
+moving code from core to addons (e.g. the [`Ember.String` deprecation](https://github.com/emberjs/rfcs/blob/master/text/0236-deprecation-ember-string.md))
+or the ["Explode RFC"](https://github.com/emberjs/rfcs/blob/explode/text/0000-explode.md). In that regard removing the 
+dependency on jQuery is a rather low hanging fruit with an high impact.
```

Comment:
Considering the implementation complexity, describing this as "low hanging fruit" seems ridiculous.

Reply:
Which implementation complexity are you referring to? For Ember itself, most of the hard work has already been done, what is proposed here should not be too hard to implement. 

If you refer to user's apps, sure that might involve a lot of work to get rid of jQuery depending on the amount of jQuery code (so "your milage may vary"). However nobody forces you to do so, and for apps with less of an jQuery legacy, removing it could be a big and rather easy win!